rosetta/validator/block.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 // Block identifier tries to extrapolate the block identifier to a full version
-// of itself. For now, we will always need a height.
+// of itself. For now, a height is always needed.
```

Comment:
Perhaps we could say something like "For now, height is mandatory." or "For now, height is required."?